core/src/org/pentaho/di/core/Const.java

```diff
@@ -1357,6 +1357,30 @@ public static String rightPad( StringBuffer ret, int limit ) {
   }
 
   /**
+   * Right pad a StringBuilder: adds spaces to a string until a certain length. If the length is smaller then the limit
+   * specified, the String is truncated.
+   *
+   * @param ret
+   *          The StringBuilder to pad
+   * @param limit
+   *          The desired length of the padded string.
+   * @return The padded String.
+   */
+  public static String rightPad( StringBuilder ret, int limit ) {
```

Comment:
How about for all these copypasta functions, we just replace them with a single method that takes a CharSequence instead?  That way we can work with String, Buffer, and Builder inputs.